tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = 
+  py36
+
+[testenv]
+deps = 
+  pytest
+  pytest-cov
+  django==2.0.5      
+
+commands =
+  {envbindir}/pytest
```

Comment:
The `{envbindir}/` here is not necessary, it works for me locally without it. I think my previous problems were due to having an installation of `tox-pipenv` still sitting around locally; once I ran `pipenv clean` things worked.

So let's remove `{envbindir}/`.